### PR TITLE
readd ping_task as keepalive for daemon

### PR DESCRIPTION
**Pull request by wjblanke**

### CODE REVIEW

**Comment by wjblanke** [comment]
Screen Shot 2022-04-28 at 12 22 13 PM

i got rid of this part, not sure if it is still needed

File: chia/daemon/server.py [hunk 1]
```diff
@@ -270,6 +272,28 @@ def remove_connection(self, websocket: WebSocketResponse):
...
+                for socket in sockets:
+                    try:
+                        self.log.info(f"About to ping: {service_name}")
+                        await socket.ping()
+                    except asyncio.CancelledError:
+                        self.log.info("Ping task received Cancel")
+                        restart = False
+                        break
+                    except Exception as e:
+                        self.log.info(f"Ping error: {e}")
```
Comment (paninaro) [inline chia/daemon/server.py:290]: self.log.exception(f"Ping error")

File: chia/daemon/server.py [hunk 2]
```diff
@@ -270,6 +272,28 @@ def remove_connection(self, websocket: WebSocketResponse):
...
+        await asyncio.sleep(30)
+        for remote_address, service_name in self.remote_address_map.items():
+            if service_name in self.connections:
+                sockets = self.connections[service_name]
+                for socket in sockets:
+                    try:
+                        self.log.info(f"About to ping: {service_name}")
+                        await socket.ping()
+                    except asyncio.CancelledError:
+                        self.log.info("Ping task received Cancel")
```
Comment (paninaro) [inline chia/daemon/server.py:286]: self.log.warning  maybe?

File: chia/daemon/server.py (hunk 1 above)
Comment (mariano54) [reply chia/daemon/server.py:290]: log.error?

**Pull request merged**